packages/fxa-payments-server/src/routes/Subscriptions/PaymentUpdateForm.tsx

```diff
@@ -76,6 +76,11 @@ export const PaymentUpdateForm = ({
     .unix(customerSubscription.current_period_end)
     .format('MM/DD/YYYY');
 
+  const expirationDate = dayjs()
+    .set("month", exp_month)
+    .set("year", exp_year)
+    .format("MMMM, YYYY");
```

Comment:
The issue didn't give us an exact format, but I'd guess we don't want a comma here.